doc/reference/peripherals/kscan.rst

```diff
@@ -0,0 +1,32 @@
+.. _kscan:
+
+
+KSCAN
+#####
+
+Overview
+********
+The kscan driver (keyboard scan matrix) is used for detecting a key press in a
+connected matrix keyboard or any device with buttons such as joysticks.
+Typically, matrix keyboards are implemented using a two-dimensional
+configuration in order to sense several keys.  This allows interfacing to
+many keys through less number of physical pins. Keyboard matrix
```

Comment:
change "less number of"  here to "fewer"  (if you can count them, you use "fewer", as in "fewer physical pins", and if you can't count them you use "less" as in "less rainfall")